contracts/core_amm/liquidity_pool.cairo

```diff
@@ -561,16 +561,6 @@ func adjust_lpool_balance_and_pool_locked_capital_expired_options{
 
     let (current_lpool_balance: Uint256) = get_lpool_balance(lptoken_address);
     let (current_locked_balance: Uint256) = get_pool_locked_capital(lptoken_address);
-    let (current_pool_position) = get_option_position(
-        lptoken_address, option_side, maturity, strike_price
-    );
-    let current_pool_position_uint256: Uint256 = toUint256_balance(current_pool_position, lpool_underlying_token);
-
-    let new_pool_position = current_pool_position - option_size;
-    with_attr error_message("new_pool_position is negative in adjust_lpool_balance_and_pool_locked_capital_expired_options") {
-        assert_nn(new_pool_position);
-    }
-    set_option_position(lptoken_address, option_side, maturity, strike_price, new_pool_position);
```

Comment:
the call to set_option_position was removed and isn't reintroduced elsewhere in this function, I understand that it was called twice before?

Reply:
it was called twice -  in this function, and right after that in the expire_option_token_for_pool with `new_pool_pos = 0`

Reply:
so the call in expire_option_token_for_pool would overwrite it anyway

Reply:
as long as you think it's tested enough (I believe pool position is tested after expiry) then LGTM